python/pyspark/sql/tests/test_catalog.py

```diff
@@ -135,7 +135,7 @@ def test_list_functions(self):
             self.assertEqual(functions, functionsDefault)
 
             with self.function("func1", "some_db.func2"):
-                spark.catalog.registerFunction("temp_func", lambda x: str(x))
+                spark.udf.register("temp_func", lambda x: str(x))
```

Comment:
to get rid of the deprecated warning in the unit tests

Reply:
nice